pdcupdater/utils.py

```diff
@@ -527,7 +527,38 @@ def pkgdb2release(collection):
     return "-".join(release)
 
 
-def tag2release(tag):
+def tag2release(tag, pdc=None):
+    if pdc:
+        return _tag2release_with_pdc(pdc, tag)
+    else:
+        return _tag2release_fedora(tag)
+
+
+def _tag2release_with_pdc(pdc, tag):
+    """ Use PDC to figure out what release is associated with this tag. """
+    releases = list(pdc.get_paged(
+        pdc['releases']._,
+        brew_allowed_tag=tag,
+    ))
+
+    if not releases:
+        raise ValueError("Could not find matching release for tag %r" % tag)
```

Comment:
"This patch lets us use that field if configured, but still retains the guesswork code for when PDC cannot provide that info." sounds contradictory with the ValueError here?

Unless you mean "when PDC cannot provide that" as "when we tell it not to use PDC", which sounds to me like you're trying to implement a new feature partially but not really?